### PR TITLE
BugFixes & Enhancements

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Bug 1059392 | Update regex and html code to accept only integer numbers in days input input field
- Bug 1059394 | Calling wrong playbook on Wizard Finish button
- Bug 1059363 | The error message is prompting for IBM qradar but not switching to the right tabs on next button of config page
- Bug 1059391  | After navigating to the investigation schedule page, upon returning to the connector configuration page, a failure occurs

Enhancements
- Remove the hardcoded values of schedule name and playbook IRI from Schedule Playbook Directive
- If Investigate_Outbreak-Alerts schedule is not present then provided the values that user want to create the schedule or not and on that basis handle the enable and disable conditions on crone tab and save button 
-

### CODE REVIEW

File: widget/view.controller.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
         $scope.triggerAutoInstallOutbreaksPlaybook = triggerAutoInstallOutbreaksPlaybook;
         $scope.backStartPage = backStartPage;
         $scope.configHuntTool = configHuntTool;
-        $scope.onlyNumbers = '^0*[1-9][0-9]*\d*$';
+        $scope.onlyNumbers = '^[1-9]\\d*$';
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:23]: only single digit number is expected?

File: widget/view.controller.js [hunk 2]
```diff
@@ -48,6 +48,10 @@
         $scope.finishInfoGraphics = widgetBasePath + 'images/finish.png';
         $scope.widgetCSS = widgetBasePath + 'widgetAssets/wizard-style.css';
         $scope.activeTab = 0;
+        $scope.ingestionDetails = {
+            "name": "Outbreak-Alerts",
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:52]: Please move this to conts

File: widget/view.controller.js [hunk 3]
```diff
@@ -228,12 +232,25 @@
                     }
                 ]
             };
-            $resource(API.QUERY + 'solutionpacks').save({ $limit: ALL_RECORDS_SIZE }, queryBody).$promise.then(function (response) {
-                if (response['hydra:member'] || response['hydra:member'].length > 0) {
-                    var huntToolDetails = _.map(response['hydra:member'], obj => _.pick(obj, ['name', 'label', 'version', 'uuid']));
-                    _loadConnectorDetails(huntToolDetails[0].name, huntToolDetails[0].version, huntToolDetails[0]);
-                }
-            });
+            $resource(API.QUERY + 'solutionpacks').save({ $limit: ALL_RECORDS_SIZE }, queryBody).$promise
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:235]: can this be part of service code?

File: widget/widgetAssets/locales/en.json [hunk 4]
```diff
@@ -74,12 +74,22 @@
         "SIXTH_PAGE_SUMMARY_HEADING_5": "Automatically install outbreak alert solution packs for outbreak alerts category: ",
         "SIXTH_PAGE_SUMMARY_HEADING_5_1": "severity",
         "SIXTH_PAGE_SUMMARY_HEADING_6": "Following email addresses will receive Outbreak alert update notifications: ",
-        "SIXTH_PAGE_AUTO_INSTALL_HEADING_1": "To install the outbreak alert solution pack now, click",
-        "SIXTH_PAGE_AUTO_INSTALL_BUTTON_LABEL": "Auto Install",
+        "SIXTH_PAGE_AUTO_INSTALL_HEADING_1": "To install the last 30 days outbreak alert solution pack now, click to ",
```
Comment (cs-suyog-jinturkar) [inline widget/widgetAssets/locales/en.json:77]: 30 is configurable?

File: widget/widgetAssets/schedulePlaybook.directive.js [hunk 5]
```diff
@@ -91,8 +123,10 @@
       function loadScheduleData() {
         $http({
           method: 'GET',
-          url: API.WORKFLOW + 'api/scheduled/?format=json&name=Investigate_Outbreak-Alerts'
+          url: API.WORKFLOW + 'api/scheduled/?format=json&name=Investigate_' + scope.jsonData.name
         }).then(function (response) {
+          scope.params.form.schedule.kwargs.exit_if_running = true;
+          scope.params.form.scheduled = 'Y'
```
Comment (cs-suyog-jinturkar) [inline widget/widgetAssets/schedulePlaybook.directive.js:129]: please use boolean here

Comment (cs-sujeet-ahirrao) [reply widget/widgetAssets/schedulePlaybook.directive.js:129]: same code used in UI code .. hence followed same terminology

File: widget/view.controller.js (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js:235]: we are not used service here so added here

Comment (cs-suyog-jinturkar) [reply widget/view.controller.js:235]: introduce one later, if same case arises

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**